pyproject.toml

```diff
@@ -1,8 +1,34 @@
 [tool.poetry]
 name = "sqlalchemy-ibmi"
-version = "0.1.0"
-description = "'SQLAlchemy support for IBM i db2'"
-authors = ["Naveen-Ram <naveen.ram@ibm.com>"]
+version = "0.9.0"
+description = "SQLAlchemy support for Db2 on IBM i"
+readme = "README.md"
+author = [
+    "Naveen Ram <naveen.ram@ibm.com>"
+    "Kevin Adler <kadler@us.ibm.com>"
+]
+license = "Apache-2.0"
+keywords = ["sqlalchemy", "database", "ibm", "ibmi", "db2"]
+classifiers =
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache License 2.0",
```

Comment:
```suggestion
```

https://python-poetry.org/docs/pyproject/#classifiers

FYI Poetry automatically adds this from the license field, so it's not needed.